src/views/TransferNft/index.tsx

```diff
@@ -21,21 +22,31 @@ export function TransferNftPage() {
         </PageCard>
 
         <div className="mt-4 w-full md:w-[500px]">
-          <Alert title="Notes for the feature" color="teal" radius="lg" className="!border !border-orange-200">
-            <div className="mb-1 color-slate-800">
-              <div>
-                Transfer NFT is a feature to help you transfer target NFT(s) to other addresses.
-                Here's the tutorial for the feature:
-              </div>
-            </div>
+          <Alert title="Helps" color="orange" radius="lg" className="!border !border-orange-200 ">
+            <ToggleCollapse title="How to transfer NFT(s)?">
+              <div className="text-slate-800">
+                <div className="mb-1">
+                  <div>
+                    Transfer NFT is a feature to help you transfer target NFT(s) to other addresses.
+                    Here's the tutorial for the feature:
+                  </div>
+                </div>
 
-            <ol className="list-decimal list-inside color-slate-900 font-semibold">
-              <li>Select an NFT that you want to transfer</li>
-              <li>Paste a L1 Wallet address to receive selected NFT(s)</li>
-              <li>Click the Submit button to start transferring</li>
-              <li>Confirm transaction and wait for the result</li>
-              <li>Keep the TxHash after submission</li>
-            </ol>
+                <ol className="list-decimal list-inside font-semibold">
+                  <li>Select an NFT that you want to transfer</li>
+                  <li>Paste a L1 Wallet address to receive selected NFT(s)</li>
+                  <li>Click the Submit button to start transferring</li>
+                  <li>Confirm transaction and wait for the result</li>
+                  <li>Keep the TxHash after submission</li>
+                </ol>
+              </div>
+            </ToggleCollapse>
+            <ToggleCollapse title="Early Accessing">
+              <div className="text-slate-800">
+                The Godwoken NFT Bridge (beta) is still in early development, you may experience issues using the app.
+                If you find any issues, please <a className="text-emerald-600 underline" href="https://github.com/godwokenrises/nft-bridge-ui/issues" target="_blank">open an issue</a> to let us know.
+              </div>
+            </ToggleCollapse>
```

Comment:
If they appear on multiple pages, would it be more appropriate to encapsulate them again?

Reply:
But for now, it doesn't seem to matter

Reply:
This alert appears in multiple pages and it has two sets of styles, so I think we can leave it like this for now. 
But if we need to edit its content next time, we should create a component for it.